core/tokens/colors.json

```diff
@@ -1,512 +1,418 @@
 {
   "colors": [
     {
-      "color": "theme",
+      "color": "blue",
```

Comment:
We are using `$md-theme-*` throughout the CSS. Removing it would break the current CSS. If you are removing it from the design system, then all of the usage instances would need to be replaced with `$md-blue-*`

Reply:
Fixed with new commit: updating core color variables